rialto_airflow/mais.py

```diff
@@ -106,24 +86,24 @@ def fetch_orcid_users(
             break
 
         params["page"] += 1  # Increment for next page
-        url = f"{base_url}/mais/orcid/v1{next_link}"  # Construct url using next link value
+        url = f"{base_url}/orcid/v1{next_link}"  # Construct url using next link value
 
     return orcid_users[:limit] if limit else orcid_users
 
 
 def fetch_orcid_user(access_token: str, base_url: str, user_id: str) -> ORCIDRecord:
     """Fetches a single ORCID user record by ORCID iD or SUNet ID from the MAIS ORCID API."""
     cleaned_id = normalize_orcid(user_id)
-    return get_response(access_token, f"{base_url}/mais/orcid/v1/users/{cleaned_id}")
+    return get_response(access_token, f"{base_url}/orcid/v1/users/{cleaned_id}")
 
 
-def current_orcid_users(access_token: str) -> list[ORCIDRecord]:
+def current_orcid_users(access_token: str, base_url: str) -> list[ORCIDRecord]:
```

Comment:
let's just pass in the base_url here like we do for every other method in this file